packages/app/app/components/LibraryView/LibraryFolderTree/index.js

```diff
@@ -0,0 +1,166 @@
+import React, {useMemo, useEffect, useRef} from 'react';
+import {useDispatch} from 'react-redux';
+import {compose} from 'recompose';
+import {withTranslation} from 'react-i18next';
+import PropTypes from 'prop-types';
+import { ContextPopup, getThumbnail } from '@nuclear/ui';
+import TrackPopupButtons from '../../../containers/TrackPopupButtons';
+import './styles.scss';
+import BaseTable, { AutoResizer } from 'react-base-table';
+import {updateExpandedFolders} from '../../../actions/local';
+
+const useTreeData = (tracks, localFolders) => {
+  return useMemo(() => {
+    const pathToEntryMap = {};
+    function getEntryForFolder(path) {
+      if (pathToEntryMap[path] === undefined) {
+        const newEntry = {
+          id: path,
+          path,
+          name: path.split('/').slice(-1)[0],
+          children: []
+        };
+  
+        if (localFolders.includes(path)) {
+          newEntry.parentId = -1;
+        } else if (path.includes('/')) {
```

Comment:
I think that the best course of action for platform compatibility would be to use `path.sep` instead of `/`

Reply:
Will do.

Reply:
Hmm, but making this change would require that the track metadata be created differently per platform. Isn't it best to standardize on "/" within the persistent metadata, rather than change it per platform? For example, using path.sep for the `track.path` property would mean you couldn't directly transfer the metadata between different-os computers.

Reply:
The persisted paths don't need to be uniform across platforms, since the directory structures won't match on different machines anyway (you won't be able to transfer the library data to a machine running a different OS). I think you're right though, let's stick with your initial idea, it's simpler.